tests/smb/smb_load.py

```diff
@@ -318,15 +325,20 @@ def delete(smbclient, load_test_config, process):
         raise OperationFailedError(f"delete operation failed, Error {e}")
 
 
-def cleanup(client, load_test_config):
+def cleanup(
+    client, smb_server, smb_share, smb_user_name, smb_user_password, load_test_config
+):
     """clean up
     Args:
         client (obj): client obj
         load_test_config (list): Smb load test parameter
     """
     try:
         load_dir = load_test_config["load_dir"]
-        cmd = f'smbclient -U user1%passwd //10.0.209.9/share1 -c "deltree {load_dir}"'
+        cmd = (
```

Comment:
@MohitBis do you think converting smbclient to lib will be considering we are going to use it most of the test cases?

Reply:
Hi @adityaramteke 

We are using "deltree" command which is specific to load test only.

pysmb not comes with per defined function for deltree

Reply:
Okay.